package.json

```diff
@@ -5,10 +5,12 @@
   "main": "index.js",
   "dependencies": {},
   "peerDependencies": {
-    "eslint-config-standard": "^4.4.0",
-    "eslint-plugin-no-console-log": "^1.0.0",
-    "eslint-plugin-react": "^3.6.2",
-    "eslint-plugin-standard": "^1.3.1"
+    "babel-eslint": "^5.0.0",
+    "eslint": "^2.1.0",
+    "eslint-config-standard": "^5.1.0",
+    "eslint-config-standard-react": "^2.3.0",
+    "eslint-plugin-react": "^4.2.3",
+    "eslint-plugin-standard": "^1.3.2"
```

Comment:
On second thought, since we have control over the tools we are using for Zanata's JavaScript builds, it is more pragmatic to keep the eslint peer dependency - it means we can specify the eslint version requirement somewhere enforceable. We can always change the config if we start using an eslint runner that does not have eslint at the top of the module hierarchy.